### PR TITLE
Translate to Chinese

### PR DESCRIPTION
**Pull request by sjfkai**
I have translated this repo to simplified Chinese.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![header](https://raw.githubusercontent.com/loverajoel/jstips/master/resources/jstips-header-blog.gif)
+![header](https://raw.githubusercontent.com/sjfkai/jstips/master/resources/jstips-header-blog.gif)
```
Comment (drakmail) [inline README.md]: Please doesn't change links to primary repo in `README.md`

Comment (sjfkai) [inline README.md]: sorry , I have fixed it.

File: README.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Any improvements or suggestions are more than welcome!
 To get updates, watch the repo and follow the [Twitter account](https://twitter.com/tips_js), only one tweet will be sent per day. It is a deal!
 > Don't forget to Star the repo, as this will help to promote the project!
 
+var entry = require('./../../dev/webpack/entry');
```
Comment (drakmail) [inline README.md]: I think this is unnecessary :)

Comment (sjfkai) [inline README.md]: sorry angin , it's a careless mistake.

**Comment by drakmail** [comment]
:+1: !

**Comment by loverajoel** [comment]
Amazing job man!!!!! I will need your help to translate statics words for the blog, I will back to you

**Comment by loverajoel** [comment]
When you finish the tip 01 and 02 I will merge this! :clap: :clap: :clap: :clap:

**Comment by microlv** [comment]
Great!!!

**Comment by mrvicadai** [comment]
nice work!

**Comment by loverajoel** [comment]
Hi @sjfkai, the structure of the repo changed and now we support multi language for the site. Please let me know when you finish to move all files to the new structure so we can merge this.
Thanks for awesome work!

**Pull request closed without merging**